aws-replicator/aws_replicator/server.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from localstack.http import Request, Response
+
+from aws_replicator.replicate import ResourceReplicator
+from aws_replicator.service_states import ReplicateStateRequest
+
+
+class RequestHandler:
+    def on_post(self, request: Request):
+        return self.__call__(request)
+
+    def __call__(self, request: Request, **kwargs) -> Response:
+        if request.method != "POST":
+            return Response(status=404)
+        content = json.loads(request.get_data(as_text=True))
+        req = ReplicateStateRequest(**content)
+        result = handle_replicate_request(req) or {}
+        return Response(json.dumps(result))
```

Comment:
since the router you are injecting this into uses the `localstack.http.dispatcher.Handler` protocol, you can just `return result`, which is better because it also sets the content type correctly to `application/json`.
alternatively you can do
`r = Response(); r.set_json(result);`